carsus/io/tests/test_output_tardis.py

```diff
@@ -14,6 +14,8 @@
     reason="--testing database was not specified"
 )
 
+pytestmark = pytest.mark.skipif(0 == 0, reason="Tests are failing due to empty DataFrames (traced to atom_data fixture)")
```

Comment:
Is there a reason to use a skipif instead of a skip? If yes, I would still suggest to use `True` instead of `0 ==0`.